shell/platform/tizen/tizen_window_elementary.h

```diff
@@ -16,16 +16,16 @@ namespace flutter {
 
 class TizenWindowElementary : public TizenWindow {
  public:
-  TizenWindowElementary(Geometry geometry,
+  TizenWindowElementary(TizenViewBase::Geometry geometry,
```

Comment:
nit:
```suggestion
  TizenWindowElementary(Geometry geometry,
```
This also applies to `TizenWindowEcoreWl2` and `TizenViewElementary`.